### PR TITLE
[COOK-1205] Add nginx support

### PR DESCRIPTION
**Pull request by chrisroberts**

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -28,7 +30,7 @@
   default['munin']['docroot'] = "/srv/http/munin"
   default['munin']['dbdir'] = "/var/lib/munin"
   default['munin']['root']['group'] = "root"
-when "centos","redhat","amazon"
+when "centos","redhat"
```
Comment (miketheman) [inline attributes/default.rb:33]: Why is this removing amazon support?

File: recipes/server_apache.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+node['apache']['default_modules'] << 'expires' if platform?("redhat", "centos", "scientific", "fedora")
```
Comment (miketheman) [inline recipes/server_apache.rb:1]: Again, amazon platform support removed.

File: attributes/default.rb (hunk 1 above)
Comment (jtimberman) [inline attributes/default.rb:33]: We can add that back after merging :)

**Comment by jtimberman** [comment]
I added the amazon platform in those lines, @miketheman. Thanks for catching that. You can find it in master, and will be included in the next release of the cookbook.

**Comment by miketheman** [comment]
Thanks, I was worried, as I added that in COOK-1122. :)

**Pull request merged**